target/linux/mvebu/image/cortexa9_neon.mk

```diff
@@ -0,0 +1,147 @@
+#
+# Copyright (C) 2012-2016 OpenWrt.org
+# Copyright (C) 2016 LEDE-project.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+define Device/cznic_turris-omnia
+  DEVICE_VENDOR := CZ.NIC
+  DEVICE_MODEL := Turris Omnia
+  KERNEL_INSTALL := 1
+  KERNEL := kernel-bin
+  KERNEL_INITRAMFS := kernel-bin
+  DEVICE_PACKAGES :=  \
+    mkf2fs e2fsprogs kmod-fs-vfat kmod-nls-cp437 kmod-nls-iso8859-1 \
```

Comment:
Based on frequent use, the indent of these hanging lines in device definitions should be a single tab.